lib/compiler.js

```diff
@@ -11,7 +11,15 @@
  */
 
 const debug = require('debug')('virtual-jade:compiler')
-const assert = require('assert')
+
+function assertWithDetailedError(node, assertion, message) {
+  if (!assertion) {
+    let err = new Error(message)
```

Comment:
thanks!  I overlooked that travis runs a different suite.  I've updated the filenames, and am working on the third case of error:  the [this shouldn't be happening error](https://github.com/pugjs/pug/blob/v1.x.x/lib/index.js#L120).